src/GraphQL.Tests/Types/AutoRegisteringObservableTests.cs

```diff
@@ -219,9 +243,17 @@ public class TestClass
         public static async Task<IObservable<string>> ListOfStrings2() => new string[] { "a", "b", "c" }.ToObservable();
         public static async ValueTask<IObservable<string>> ListOfStrings3() => new string[] { "a", "b", "c" }.ToObservable();
         [OutputType(typeof(StringGraphType))]
-        public static IObservable<object> ListOfStrings4() => new string[] { "a", "b", "c" }.ToObservable();
+        public static IObservable<object> ListOfStrings4() => new object[] { "a", "b", "c" }.ToObservable();
         [OutputType(typeof(StringGraphType))]
-        public static async ValueTask<IObservable<string>> ListOfStrings5() => new string[] { "a", "b", "c" }.ToObservable();
+        public static async ValueTask<IObservable<object>> ListOfStrings5() => new object[] { "a", "b", "c" }.ToObservable();
```

Comment:
This test was incorrect and has been fixed (ListOfStrings3 already tests this scenario).

Reply:
One more combination - `Task<IObservable<object>>`

Reply:
done